app/controllers/sessions_controller.rb

```diff
@@ -4,6 +4,13 @@ def new
   end
 
   def create
+    if request.env['omniauth.auth']
+      user = User.create_with_omniauth(request.env['omniauth.auth'])
+      session[:user_id] = user.id
+      user.save!
+      flash[:notice] = "You are now logged in as #{user.first_name}"
+      redirect_to "/profile" 
+    else 
```

Comment:
Nice job getting this working Sage!
Let's definitely remember to talk to Dione about any advice she on the password confirmation on Monday

Reply:
Great work here sage!